Gemfile

```diff
@@ -34,10 +34,15 @@ gem 'mini_magick'
 
 # Payment processing
```

Comment:
Exactly, especially with larger projects the Gemfile becomes a bizarre collection of gems you have no idea what each one does.